cluster/deploy-gpu-player.sh

```diff
@@ -16,4 +16,6 @@
 
 source ./common.sh
 
+# TODO(kashomon): envsubst doesn't exist for OSX. needs to be brew-installed
+# via gettext. Should probably warn the user about that.
```

Comment:
in the README?  Otherwise this comment is pretty much all that you need :)

Reply:
Added it directly in the command